data/campaigns/World_Conquest/Modding.md

```diff
@@ -0,0 +1,291 @@
+# Modding World Conquest
+
+This section describes how to make eras/modifications to manipulate World Conquest elements.
+
+## How to add an era's units to World Conquest's enemy selection pool
+
+> This file describes how to make your addon work well together with world conquest 2. In particular how to write an era that can be used together with world conquest ii.
```

Comment:
Should all of the "world conquest 2" and "world conquest ii" lose the number?

There's a "word" conquest typo on line 11 too.

Reply:
> Should all of the "world conquest 2" and "world conquest ii" lose the number?

That was the idea when adding WC to mainline, yes.

There is no rush in converting internal variables etc but public API should definitely be fixed.